internal/local/info.go

```diff
@@ -158,28 +218,44 @@ func (service *ProvisioningService) DisplayAMTInfo() (err error) {
 		if err != nil {
 			log.Error(err)
 		}
+
+		wired_osIpAddress, err := GetOSIPAddress(wired.MACAddress)
+		if err != nil {
+			log.Error(err)
+		}
+		wired.OsIPAddress = wired_osIpAddress
+
 		dataStruct["wiredAdapter"] = wired
 
 		if wired.MACAddress != "00:00:00:00:00:00" {
 			service.PrintOutput("---Wired Adapter---")
 			service.PrintOutput("DHCP Enabled 		: " + strconv.FormatBool(wired.DHCPEnabled))
 			service.PrintOutput("DHCP Mode    		: " + wired.DHCPMode)
 			service.PrintOutput("Link Status  		: " + wired.LinkStatus)
-			service.PrintOutput("IP Address   		: " + wired.IPAddress)
+			service.PrintOutput("AMT IP Address   	: " + wired.IPAddress)
```

Comment:
check spacing on the `:` to ensure its aligned

Reply:
change the space to tab